packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -97,10 +104,14 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
 
       const originPolicy = location.originPolicy
 
+      // This is not reset after leaving the switchToDomain command.
       cy.state('latestActiveOriginPolicy', originPolicy)
+      // This is set while IN the switchToDomain command.
+      cy.state('currentActiveOriginPolicy', originPolicy)
```

Comment:
I wasn't thrilled about adding this but they are serving different purposes, `current` indicates what the origin policy is for the currently running  switchToDomain command where as `latest` can be uses after the switchToDomain command has completed to track onLoad events.